examples/llama/generator.py

```diff
@@ -169,13 +214,20 @@ def log(msg):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("--pth_path", type=str, required=True)
+    parser.add_argument("--ckpt_dir", type=str, required=True)
     parser.add_argument("--params_path", type=str, required=True)
     parser.add_argument("--tok_path", type=str, required=True)
     parser.add_argument("--ngpus", type=int, default=1)
+    parser.add_argument("--only_torch_model", type=bool, default=False)
 
     args = parser.parse_args()
 
+    os.environ["ARK_IPC_LISTEN_PORT_BASE"] = "42500"
+
+    # For torch.distributed
```

Comment:
Do we use `torch.distributed`?

Reply:
Add this to run Llama2 official model. Maybe I'd better create another file to run this one